api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -438,6 +439,17 @@ default void hideBossBar(final @NonNull BossBar bar) {
   default void playSound(final @NonNull Sound sound) {
   }
 
+  /**
+   * Plays a sound at a location.
+   *
+   * @param sound a sound
+   * @param location a location
+   * @since 4.8.0
+   */
+  default void playSound(final @NonNull Sound sound, final @NonNull Vector3dLike location) {
+    this.playSound(sound, location.x(), location.y(), location.z());
```

Comment:
```suggestion
   * Plays a sound at a position.
   *
   * @param sound a sound
   * @param pos a position
   * @since 4.8.0
   */
  default void playSound(final @NonNull Sound sound, final @NonNull Vector3dLike pos) {
    this.playSound(sound, pos.x(), pos.y(), pos.z());
```